R/local.R

```diff
@@ -101,7 +101,11 @@ local_reproducible_output <- function(width = 80,
     rlang_interactive = FALSE,
     .local_envir = .env
   )
-  withr::local_envvar(RSTUDIO = NA, .local_envir = .env)
+  withr::local_envvar(
+    RSTUDIO = NA,
+    LANGUAGE = "en",
```

Comment:
Should we include a sentence about this in the list of things `local_test_context()` sets, up about line 44? 

Btw. that line also needs a fix for `RSTUDIO`, which is now unset.